Changelog v1.8.x.md

```diff
@@ -3,14 +3,20 @@
 - New: More reinforced block recipes for parity with vanilla (Thanks Redstone_Dubstep!)
 - New: Fake Water/Lava Buckets can now be properly used in dispensers
 - New: Add all ownable blocks to DRAGON_IMMUNE and WITHER_IMMUNE block tags
+- New: Trophy System now destroys shulker bullets, dragon fireballs and wither skulls
+- Change: Bouncing Betty now jump as high as the player's head
 - Fix: Crash when disguising a block with itself
 - Fix: Localization for camera keybindings is incorrect
 - Fix: Incorrect laser removal in creative mode
 - Fix: Reinforced Glowstone and Reinforced Sea Lantern don't give off light (Thanks Redstone_Dubstep!)
 - Fix: German language fixes (Thanks Redstone_Dubstep!)
 - Fix: Incorrect version check with VersionChecker mod integration
-- Fix: Cage Trap owner collision is incorrect
+- Fix: Cage Trap collision is incorrect
 - Fix: Inventory Scanners can connect to Inventory Scanners by different owners
+- Fix: Bouncing Betty doesn't play a sound when triggered
+- Fix: Block Mines let items and the owner go through
+- Fix: Trophy System destroys projectile even if it has already been removed
+- Fix: Textures of Universal Block Modifier, Universal Owner Changer and the Mine Remote Access tool are transparent at the sides
```

Comment:
```suggestion
- Fix: Textures of Universal Block Modifier, Universal Owner Changer and the Mine Remote Access Tool are transparent at the sides
```

I don't know of any other items at the moment, if you want you can go through the items of the mod, or we'll fix them as we find them.